File: app/src/androidTest/java/com/jpmorgan/typicode/utils/TestUtils.java
package com.jpmorgan.typicode.utils;

import com.jpmorgan.typicode.pojos.Album;

public class TestUtils {

    public static Album[] getAlbums() {

        Album album = new Album();
        album.setTitle("Dr Who");
        album.setUserId(1000000);
        album.setId(21);
        return new Album[]{album};
    }
}

File: app/src/androidTest/java/com/jpmorgan/typicode/utils/TestUtils.kt
package com.jpmorgan.typicode.utils

import com.jpmorgan.typicode.pojos.Album

object TestUtils {

    val albums: Array<Album>
        get() {

            val album = Album()
            album.title = "Dr Who"
            album.userId = 1000000
            album.id = 21
            return arrayOf(album)
        }
}
